### ISSUE TITLE
Error happens when a JDownloader device that is down is configured

### ISSUE THREAD

**Issue by cremor** [piejanssens/premiumizer#174]
If you configure a JDownloader device in the settings that is not currently running clicking on "Save" shows an "Internal Server Error" page.

I'm using the Docker image that was build for commit ad2b8ced3d1ed8c4eb11066e2370e8eb3b64e09f

Log:

```
03-23 10:21:16: ERROR : Exception on /settings [POST]
Traceback (most recent call last):
  File "./premiumizer/premiumizer.py", line 325, in check_config
    self.jd_device = self.jd.get_device(self.jd_device_name)
  File "/usr/local/lib/python3.7/site-packages/myjdapi/myjdapi.py", line 963, in get_device
    raise (MYJDException("Device not found\n"))
myjdapi.myjdapi.MYJDException: Device not found


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python3.7/site-packages/flask_login/utils.py", line 258, in decorated_view
    return func(*args, **kwargs)
  File "./premiumizer/premiumizer.py", line 2127, in settings
    cfg.check_config()
  File "./premiumizer/premiumizer.py", line 330, in check_config
    logger.error('myjdapi : ' + str(e.message))
AttributeError: 'MYJDException' object has no attribute 'message'
```

**Comment by neox387** [comment]
Fixt with next pr.

THX =D

**Comment by cremor** [comment]
Closing this since it was fixed in #175
